src/unix/notbsd/linux/other/mod.rs

```diff
@@ -670,8 +670,7 @@ pub const NFPROTO_NETDEV: ::c_int = 5;
 
 // linux/netfilter/nf_tables.h
 cfg_if!{
-    if #[cfg(any(target_arch = "arm", target_arch = "powerpc",
-                 target_arch = "powerpc64", target_arch = "aarch64"))] {
```

Comment:
Why this change?

Reply:
It's because of the kernel upgrade, see here:
- Ubuntu 17.10 with 4.13.x kernel: https://elixir.bootlin.com/linux/v4.13.16/source/include/uapi/linux/netfilter/nf_tables.h
- Ubuntu 18.04 with 4.15.x kernel: https://elixir.bootlin.com/linux/v4.15.18/source/include/uapi/linux/netfilter/nf_tables.h